### DISCUSSION TITLE
Missing Media on Federated Posts

### DISCUSSION THREAD

**Discussion by maltechx** [mastodon/mastodon#18407] [Q&A] [answered]
Hi, 

I don't know why, but for some reason it happens that media attachments are missing randomly. It can also happen that on a post with 3 images one is missing.
I have already looked through sidekiq's logs but could not find any errors that could relate to this phenomenon. Just recently I migrated my media storage to Backblaze, but I also could not find any S3 errors.

Does anyone have any idea what the problem might be here or what I should be looking for in the logs?

Thank you, Malte

**Comment by maltechx** [comment, marked as answer]
Has been resolved. I needed to use:
```
S3_OPEN_TIMEOUT=20
S3_READ_TIMEOUT=20
```
to increase the S3 timeout.

**Reply by hermann-san** [reply]
thanks for sharing the solution
it would be great if you could mark the topic as solved
